src/GLM.jl

```diff
@@ -54,14 +55,14 @@ module GLM
         updateμ!,      # update the response type from the linear predictor
         wrkresp         # working response
 
-    typealias FP AbstractFloat
-    typealias FPVector{T<:FP} AbstractArray{T,1}
+    const FP = AbstractFloat
+    @compat const FPVector{T<:FP} = AbstractArray{T,1}
```

Comment:
I think it's the preferred style, as it's what's suggested by the deprecation warning and is what's used in Base.